packages/react-components/react-input/src/components/Input/useInputStyles.ts

```diff
@@ -217,7 +217,7 @@ const useContentStyles = makeStyles({
     boxSizing: 'border-box',
     color: tokens.colorNeutralForeground3, // "icon color" in design spec
     // special case styling for icons (most common case) to ensure they're centered vertically
-    '> svg': { display: 'block' },
+    '> svg': { display: 'flex' },
```

Comment:
I think I'm not following how this fixes #26057 -- the issue seems to be with the specificity of the `> svg` selector rather than the `display` value. When I recreate Ling's codesandbox example with this change, `bundleIcons` still shows both icons.

Are you getting a different result? Or is this for a different issue?

Reply:
I should have removed the svg styling completely and moved the display:flex styling to the base, since that is what is breaking the svg specificity. I've updated the PR now to reflect this